Assets/Scripts/UI/StatBar.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StatBar : MonoBehaviour
+{
+    public string SpriteLocation = "Sprites/PlayerHungerStates_Sprite";
+    
+    private int _statusValue = 100;
+
+    private Sprite[] _barStates;
+
+    private Image[] _images;
+
+    private Animator _animator;
+    
+    
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Please define an access modifier. I know it's not needed but it looks nicer and you know, Consistency 🎉